definitions/npm/leaflet_v1.1.x/flow_v0.50.x-v0.51.x/leaflet_v1.1.x.js

```diff
@@ -0,0 +1,339 @@
+declare module 'leaflet' {
+    declare type CornerType = LatLng | [number, number];
```

Comment:
Can you please do not use definitions hoisting? Firstly primitives, after composited types and classes. Will be better for readability.